iroha/test_network/src/lib.rs

```diff
@@ -114,12 +114,20 @@ pub const CONFIGURATION_PATH: &str = "tests/test_config.json";
 pub const CLIENT_CONFIGURATION_PATH: &str = "tests/test_client_config.json";
 pub const GENESIS_PATH: &str = "tests/genesis.json";
 
-pub fn test_arguments(submit_genesis: bool) -> Arguments {
-    Arguments {
-        submit_genesis,
-        genesis_path: GENESIS_PATH.into(),
-        config_path: CONFIGURATION_PATH.into(),
-        ..Arguments::default()
+pub trait TestGenesis: Sized {
+    fn test(submit_genesis: bool) -> Option<Self>;
+}
+
+impl<G: GenesisNetworkTrait> TestGenesis for G {
+    fn test(submit_genesis: bool) -> Option<Self> {
+        let cfg = Configuration::test();
+        G::from_configuration(
+            submit_genesis,
+            GENESIS_PATH,
+            &cfg.genesis_configuration,
+            cfg.sumeragi_configuration.max_instruction_number,
+        )
+        .expect("Failed to")
```

Comment:
Also I think as you mentioned - the purpose was not to load it from file in tests, right?

Reply:
The purpose was not to load to much files from path. I guess we can read files during compilation of crate. What are your thoughts?

Reply:
I see that this way it reads file earlier, which is good if this helps with tests.

Reply:
My initial thoughts were that you would supply genesis Instructions directly in the code. This might be useful for tests and I think our Java client team does this.

Reply:
Dont see a difference between these 2 things, apart from ide autocompletion.

Reply:
So that we will not create json for each test that wants custom genesis is a big plus to my mind.

Reply:
But anyway we can merge as is. It is just a possible improvement for the future.